storagenode/storagenodedb/bandwidthdb.go

```diff
@@ -22,9 +22,9 @@ type bandwidthdb struct {
 }
 
 type bandwidthUsed struct {
+	used      int64
```

Comment:
Add a comment why it's in a different order.

Reply:
done